app/controllers/finders_controller.rb

```diff
@@ -49,14 +49,14 @@ def content_item
   end
 
   def results
-    @results ||= result_set_presenter_class.new(finder, filter_params, view_context, show_top_result?)
+    @results ||= result_set_presenter_class.new(finder, filter_params, view_context, sort_presenter, show_top_result?)
```

Comment:
Could we somehow avoid having result_set_presenter and the finder_presenter depend on the sort_presenter?

Reply:
Yes, though it might be worth waiting for Andy's change to the templates to complete.

The result set presenter is responsible for sending a hash to the frontend as a response to the Ajax call to the `.json` endpoint.

As we now update the sort options (using the sort presenter) via this same route we need to give the sort presenter (or at least it's data) to the result set presenter.

Once Andy's template changes have been made perhaps we could make some changes to the relationship between presenters and their templates?